cms/templates/cms/toolbar/dragbar.html

```diff
@@ -25,6 +25,7 @@
 
     <div class="cms-dragbar-title">
         {{ placeholder.get_label }}
+        {% if placeholder.is_inherit %}<small>{% trans "Inherit Placeholder" %}</small>{% endif %}
```

Comment:
@czpython You're right. I can use the same pin icon like the static placeholders, and additionally we can add a tooltip about the placeholder is inherit. Does it make sense?